health-services/project-factory/src/server/utils/onGoingCampaignUpdateUtils.ts

```
@@ -0,0 +1,806 @@
+        id: projectId,
+        tenantId: tenantId
+      }
+    ]
+  }
+  const projectSearchParams = {
+    tenantId: tenantId,
+    offset: 0,
+    limit: 1,
+    includeDescendants: true
+  }
+  logger.info("Project search params " + JSON.stringify(projectSearchParams))
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)